css/index.css

```diff
@@ -238,24 +271,63 @@ label {
     right: 0;
   }
 
+  .direction-buttons {
+    font-size: smaller;
+    position: absolute;
+    top: 20px;
+    right: 15px;
+    display: inline;
+    list-style: none;
+
+    margin: 0;
+  }
+
   .welcome {
     padding-top: 100px;
     padding-bottom: 100px;
-    font-size: large;
+    font-size: medium;
+    justify-content: center;
   }
 
   .about-body,
   .xp {
     margin-left: 20px;
     margin-right: 20px;
     padding: 20px;
+    justify-content: center;
+  }
+
+  #skills {
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+    justify-content: center;
+    text-align: center;
+    padding-right: 0;
+  }
+
+  #skills ul {
+    padding: 20px;
   }
 
   .connect-images {
     left: 10%;
+    justify-content: center;
+  }
+
+  .leave_message fieldset {
+    justify-content: center;
+  }
+
+  #footer {
+    font-size: small;
+    display: inline;
+    justify-content: center;
+    margin: 0;
+    left: 10px;
   }
 }
-/* View style for tablets */
+/* View style for tablets/ipads */
```

Comment:
A couple of the sections are causing some responsiveness problems: 

<img width="272" alt="Screen Shot 2023-09-19 at 11 51 11 PM" src="https://github.com/CristianMoran1/freyr-intro/assets/12771651/2ddeee2b-3462-4b7c-84b9-dd32dc767c90">